### PR TITLE
Add Tradisional Chinese

### PR DESCRIPTION
**Pull request by chongx1an**

### CODE REVIEW

**Comment by dacap** [comment]
Thanks for your contribution @chongx1an I'll make a little comment in the commit

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ distributed as extensions (pull requests are welcome):
 * [Japanese](http://wikiwiki.jp/aseprite/?%C6%FC%CB%DC%B8%EC%B2%BD%A5%D5%A5%A1%A5%A4%A5%EB%A4%CE%A5%C0%A5%A6%A5%F3%A5%ED%A1%BC%A5%C9)
 * [Korean](https://imbada.github.io/Aseprite-Korean/) ([Korean alternative](http://eternalworld.tistory.com/531))
 * [Simplified Chinese](https://steamcommunity.com/sharedfiles/filedetails/?id=1333477949)
+* [Tradisional Chinese](https://github.com/chongx1an/aseprite-TradisionalChineseExtension)
```
Comment (dacap) [inline README.md]: I think "Traditional Chinese" would be the correct term on English, right?

Comment (chongx1an) [reply README.md]: Oh yeah, sorry for typo error.

Comment (dacap) [reply README.md]: No problem, you can make another commit or amend the one you have, whatever you choose is fine, I'll squash the commits in one before merging anyway.

**Comment by dacap** [comment]
Great! 👍 I'll merge it right now, thanks for your contribution @chongx1an

File: README.md (hunk 1 above)
Comment (chongx1an) [reply README.md]: Done the change

**Comment by chongx1an** [comment]
You are welcome :smile:

**Pull request merged**